packages/web/app/views/programRegistry/DeleteProgramRegistryFormModal.jsx

```diff
@@ -55,17 +50,31 @@ export const DeleteProgramRegistryFormModal = ({ patientProgramRegistration, onC
   };
 
   return (
-    <Modal title="Delete record" width="sm" open={open} onClose={onClose} overrideContentPadding>
+    <Modal
+      title={
+        <TranslatedText
+          stringId="patientProgramRegistry.modal.deleteRegistry.title"
+          fallback="Delete record"
+        />
+      }
+      width="sm"
+      open={open}
+      onClose={onClose}
+      overrideContentPadding
+    >
       <Text>
-        <p className="header">Confirm patient registry deletion</p>
+        <p className="header">
+          <TranslatedText
+            stringId="patientProgramRegistry.modal.deleteRegistry.header"
+            fallback="Confirm patient registry deletion"
+          />
+        </p>
         <p className="desc">
-          Are you sure you would like to delete the patient from the{' '}
-          <TranslatedReferenceData
-            fallback={patientProgramRegistration?.programRegistry?.name}
-            value={patientProgramRegistration?.programRegistry?.id}
-            category="programRegistry"
+          <TranslatedText
+            stringId="patientProgramRegistry.modal.deleteRegistry.description"
+            fallback="Are you sure you would like to delete the patient from the :name? This will delete associated patient registry records. This action is irreversible."
```

Comment:
Nitpicking but possibly

```suggestion
            fallback="Are you sure you would like to delete the patient from the :programRegistry? This will delete associated patient registry records. This action is irreversible."
```